rosbag2_converter_default_plugins/test/rosbag2_converter_default_plugins/cdr/test_cdr_converter.cpp

```diff
@@ -17,16 +17,20 @@
 #include <memory>
 #include <string>
 
-#include "../../../src/rosbag2_converter_default_plugins/cdr/cdr_converter.hpp"
-#include "../../../src/rosbag2_converter_default_plugins/logging.hpp"
 #include "rcutils/strdup.h"
-#include "rosbag2/converter_interfaces/serialization_format_converter.hpp"
-#include "rosbag2/typesupport_helpers.hpp"
-#include "rosbag2/types/introspection_message.hpp"
+
+#include "rosbag2_cpp/converter_interfaces/serialization_format_converter.hpp"
+#include "rosbag2_cpp/typesupport_helpers.hpp"
+#include "rosbag2_cpp/types.hpp"
+
 #include "rosbag2_test_common/memory_management.hpp"
+
 #include "test_msgs/message_fixtures.hpp"
 
-using rosbag2::converter_interfaces::SerializationFormatConverter;
+#include "../../../src/rosbag2_converter_default_plugins/cdr/cdr_converter.hpp"
+#include "../../../src/rosbag2_converter_default_plugins/logging.hpp"
```

Comment:
This is probably for another PR, but shouldn't these be found via the `CMakeLists.txt`?

Reply:
This is ok since the package's structure is known to the tests, but it can be fragile. You could create a relative path in CMake and pass it as a compiler definition instead. At least then it wouldn't be hardcoded in so many places. But the use of a relative path here is in principle safe.